torch/linalg/__init__.py

```diff
@@ -2320,20 +2321,17 @@
   When :attr:`mode`\ `= 'r'`, the `Q` tensor is an empty tensor.
   This behavior may change in a future PyTorch release.
 
-.. note:: The elements in the diagonal of `R` are not necessarily positive.
+.. note:: The only :attr:`mode` that is differentiable is :attr:`mode`\ `= 'reduced'`.
 
```

Comment:
Shouldn't `mode=complete` with `m < n` be differentiable as well?

Reply:
Yeah, the thing is that if `m < n`, mode complete and reduced are equivalent, so I think it's fine not advertising that really. You should be using `reduced`mode anyway.

Reply:
cc @mruberry , @IvanYashchuk . What is your take on this one?

Reply:
I'm leaning towards being explicit about what's supported if we started writing this note, i.e. I would include that supported modes are "reduced" or "complete" && `m <= n`.

Reply:
Erring on the side of more detail sounds fine (and maybe an opportunity to suggest a best practice)